R/pr.R

```diff
@@ -0,0 +1,137 @@
+pr_create <- function(branch) {
+  check_uses_github()
+
+  cur_branch <- proj_git_branch_get()
+  if (cur_branch$name == branch) {
+    return(invisible(TRUE))
+  }
+
+  pr_branch <- proj_git_branch_find(branch)
+  if (is.null(pr_branch)) {
+    if (cur_branch$name != "master") {
+      if (nope("Start pull-request from non-master branch?")) {
```

Comment:
Wording quibble: I feel like you do usually want to PR from a non-master branch. This is really about whether you want the PR branch to have a non-`master` branch as parent.